### ISSUE TITLE
Android App reporting "No Files Here" when desktop and server/web clients are both populated

### ISSUE THREAD

**Issue by oucil** [nextcloud/android#11449]
### ⚠️ Before posting ⚠️

- [X] This is a **bug**, not a question or an enhancement.
- [X] I've [searched for similar issues](https://github.com/nextcloud/android/issues) and didn't find a duplicate.
- [X] I've written a clear and descriptive title for this issue, not just "Bug" or "Crash".
- [X] I agree to follow Nextcloud's [Code of Conduct](https://nextcloud.com/contribute/code-of-conduct/).

### Steps to reproduce

This behaviour has been reported in the iOS app as well here: https://github.com/nextcloud/ios/issues/2321

Oddly for me this is only occuring on a 2nd signed in server, the first server responds normally. 

1. Add a second account login to a different Nextcloud server
2. Try to view files in root, app responds "No Files Here"
3. Try "Media" or "Activities" and app responds with content as expected.

### Expected behaviour

I expect to see the files that other parts of the app are reporting do indeed exist, as confirmed in the Desktop and Server/Web console.

### Actual behaviour

App will only respond "No Files Here". I have several android devices and they all exhibit the same behaviour.

### Android version

13 - Build #: TQ1A.230205.002

### Device brand and model

Google Pixel 7

### Stock or custom OS?

Stock

### Nextcloud android app version

3.24.1

### Nextcloud server version

25.0.4

### Using a reverse proxy?

No

### Server error logs

```bash
No error logs.
```

### Additional information

No errors reported anywhere. Both servers are brought up with the same Ansible script and have  *exactly* the same settings, with the exception of the domain. They are separate machines but are otherwise identicle.

**Comment by oucil** [comment]
Still happening in 3.25.0

**Comment by joshtrichards** [comment]
Hi @oucil - Can you install a second instance of the client and see if this happens on the second account when it's configured as, well, the only account in the client? 

The easiest way to do this would probably be to install F-Droid and the Nextcloud Dev client from there. You don't need to configure anything that will be invasive (e.g. auto-upload with delete). Just add the account and see if your files are visible.

It also appears the iOS user reinstalled their server and fixed their problem - which points at a server-side/config matter: https://github.com/nextcloud/ios/issues/2321#issuecomment-1763408507

**Comment by joshtrichards** [comment]
For the record, other than sharing the same symptoms as that report in the ios repository, there can't be any connection (at least on the client code side). This is because the Android and iOS clients don't share any common code.

**Comment by oucil** [comment]
> Can you install a second instance of the client and see if this happens on the second account when it's configured as, well, the only account in the client?

I'll give this a try, but I can already confirm that everything works fine when using both "SolidExplorer" and it's Nextcloud connectivity features, and "nc-photos" v62.0 which is photo gallery specific, but nonetheless has no problems browsing both servers.

**Comment by oucil** [comment]
Funny, just went to double check my current NC Client version (v3.26.0) and the problem is fixed, both servers are showing files now. As I reported above 3.25 wasn't working. I've upgraded the servers (all 10 in a batch) a few times in there, so it very likely could have been server side.

In any case, thanks for drawing my attention back to the issue to see the fix, after finding SolidExplorer, I stopped trying with the NC client.

Appreciate the help!
